_components/pages/Docs.tsx

```diff
@@ -0,0 +1,424 @@
+              className="dev-tools-img"
+            ></img>
+          </div>
+          <div className="col">
+            <div className="d-flex flex-column-reverse w-100">
+              <h4 className="mb-10">Explore, Test, Verify</h4>
+              <h6 className="mb-3">Explore Dev Tools</h6>
+            </div>
+            <p className="mb-20">
+              Use these web-based tools to assist during all stages of
+              development, from getting your first payment to testing your
+              implementation for best practices.{" "}
+            </p>
+            <div className="card-grid card-grid-2xN">
+              {devTools.map((card) => (
+                <DevToolsCard props={card} />
+              ))}
+            </div>
+            <a
+              className="btn btn-primary btn-arrow"
+              href="https://xrpl.org/dev-tools.html"
+            >
+              View All tools
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <section className="py-26 px-20 text-left">
+        <div className="card-grid card-grid-2xN">
+          <div className="col">
+            <h4>Browse By Recommended Pages</h4>
+            <ul className="nav flex-column">
+              {recommendedPages.map((r) => (
+                <NavLinks props={r} />
+              ))}
+            </ul>
+          </div>
+
+          <div className="col">
+            <div className="col-lg-12 p-6-sm p-10-until-sm br-8 cta-card">
+              <img
+                src={prefix + "/img/docs/cta-docs-purple.svg"}
+                className="d-none-sm cta cta-top-left"
+              />
+              <img
+                src={prefix + "/img/docs/cta-docs-green.svg"}
+                className="cta cta-bottom-right"
+              />
+              <div className="z-index-1 position-relative">
+                <h2 className="h4 mb-8-sm mb-10-until-sm">
+                  Have an Idea For a Tool?
+                </h2>
+                <p className="mb-10">
+                  Connect to the XRP Ledger Testnet network to develop and test
+                  your apps built on the XRP Ledger, without risking real money
+                  or impacting production XRP Ledger users.
+                </p>
+                <a
+                  className="btn btn-primary btn-arrow-out"
+                  href="https://xrpl.org/xrp-testnet-faucet.html"
+                >
+                  Generate Testnet Credentials
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <Helmet>
+        <script src={prefix + "/js/video.js"}></script>
+      </Helmet>
+    </div>
+  );
+}
+
+function DevToolsCard({ props }) {
+  return (
+    <div className="col">
+      <h6>
+        {props.title}
+        <a className="btn-arrow-purple" target="_blank" href={props.link}></a>
+      </h6>
+      <p> {props.description}</p>
+    </div>
+  );
+}
```

Comment:
Right now, only the little tiny arrow after the text is a hyperlink. That seems wrong. I think the whole card should be a link. For example:

```suggestion
function DevToolsCard({ props }) {
  return (
    <a className="col" target="_blank" href={props.link}>
      <h6 className="btn-arrow-purple">
        {props.title}
      </h6>
      <p>{props.description}</p>
    </a>
  );
}
```